autorest/codegen/templates/templates_package/setup.py.jinja2

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import re
+from setuptools import setup, find_packages
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "{{ package_name }}"
+PACKAGE_PPRINT_NAME = "{{ package_pprint_name }}"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace("-", "/")
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, "_version.py"), "r") as fd:
+    version = re.search(
+        r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE
+    ).group(1)
+
+if not version:
+    raise RuntimeError("Cannot find version information")
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description="Microsoft {{ package_pprint_name }} Client Library for Python",
+    long_description=open("README.md", "r").read(),
+    long_description_content_type="text/markdown",
+    license="MIT License",
+    author="Microsoft Corporation",
+    author_email="azpysdkhelp@microsoft.com",
+    url="https://github.com/Azure/azure-sdk-for-python/tree/main/sdk",
+    classifiers=[
+        "Development Status :: 4 - Beta",
```

Comment:
I think it's fair (at least for now) to only generate beta, if we're following the whole "generate packaging tools for your 1.0.0b1 release", @lmazuel for his thoughts

Reply:
If that, we need to update release pipeline since we can decide version number only when release. So the discussion is not about autorst.python but release pipeline. I think we could discuss about it this Wednesday.

Reply:
same comment with https://github.com/Azure/autorest.python/pull/1154#discussion_r810395123